src/main/java/net/sf/jabref/gui/event/listener/TestEventListener.java

```diff
@@ -0,0 +1,19 @@
+package net.sf.jabref.gui.event.listener;
+
+
+import net.sf.jabref.gui.event.Event;
+
+public class TestEventListener implements EventListener {
```

Comment:
How does the TestEventListener signals that it only wants to listen for testevents and not arbitrary events?